### ISSUE TITLE
cimport fails to escape characters correctly for zig

### ISSUE THREAD

**Issue by daurnimator** [ziglang/zig#2611]
e.g. I have a header file containing:
```C
#define FOO	"\001"
```

Trying to `@cImport()` this results in:
```
cimport.zig:177:24: error: newline not allowed in string literal
pub const FOO = c"\x1";
                       ^
```

**Comment by timonkruiper** [comment]
This can be closed: fixed with PR #2615.

**Comment by nrdmn** [comment]
I can't reproduce this issue. That's why I changed my PR so it doesn't close it.

**Comment by daurnimator** [comment]
Fixed via #2615
